test/middleware/jsonld.test.ts

```diff
@@ -1,105 +1,127 @@
+import { literal, namedNode, quad } from '@rdfjs/data-model';
 import { parse as parseContentType } from 'content-type';
-import { Context as JsonLdContext } from 'jsonld/jsonld-spec';
-import { Context, Response } from 'koa';
+import 'jest-rdf';
+import { addAll } from 'rdf-dataset-ext';
+import { Quad } from 'rdf-js';
+import { AppContext } from '../../src/app';
 import jsonld from '../../src/middleware/jsonld';
-import createContext from '../context';
-import runMiddleware, { Next } from '../middleware';
+import { rdf, schema } from '../../src/namespaces';
+import createContext, { Headers } from '../context';
+import { Next } from '../middleware';
 
-const makeRequest = async (next?: Next, jsonLdContext: JsonLdContext = {}): Promise<Response> => (
-  runMiddleware(jsonld(jsonLdContext), createContext(), next)
-);
+const makeRequest = async (
+  body?: string, headers?: Headers, next: Next = jest.fn(), jsonLdContext = {},
```

Comment:
given the potential `context`/`jsonLdContext` confusion, could the latter have a better default, or type, to immediately make it clear what it is?